proposals/compatibility-mode.md

```diff
@@ -95,6 +95,40 @@ Use of the `sample_mask` builtin would cause a validation error at pipeline crea
 
 **Justification**: OpenGL ES 3.1 does not support `gl_SampleMask` or `gl_SampleMaskIn`.
 
+### 8. Lower limits.
+
+The differences in limits between compatibility mode and standard WebGPU
+are as follows
+
+
+| limit                               | compat  | standard  |
+| :---------------------------------- | ------: | --------: |
+| `maxColorAttachments`               |       4 |         8 |
+| `maxComputeInvocationsPerWorkgroup` |     128 |       256 |
+| `maxComputeWorkgroupSizeX`          |     128 |       256 |
+| `maxComputeWorkgroupSizeY`          |     128 |       256 |
+| `maxInterStageShaderVariables`      |      15 |        16 |
```

Comment:
Is this `min(GL_MAX_VARYING_COMPONENTS, GL_MAX_FRAGMENT_INPUT_COMPONENTS, GL_MAX_VERTEX_OUTPUT_COMPONENTS) // 4`?

Reply:
This is GL_MAX_VARYING_VECTORS

Reply:
I see, but does `GL_MAX_VARYING_VECTORS` have to be equal to `min(GL_MAX_VARYING_COMPONENTS, GL_MAX_FRAGMENT_INPUT_COMPONENTS, GL_MAX_VERTEX_OUTPUT_COMPONENTS) // 4`?

Reply:
reading the spec, my understanding is GL_MAX_FRAGMENT_INPUT_COMPONENTS and GL_MAX_VERTEX_OUTPUT_COMPONENTS are always more than GL_MAX_VARYING_COMPONENTS
so GL_MAX_VARYING_COMPONENTS will always be the min value.

I think what the spec is trying to do is allow an implementation to signal that `gl_PointSize`, `gl_FragCoord`, `gl_FrontFacing`, `gl_FragDepth`, `gl_PointCoord`, `gl_HelperInvocation` will not count against your varying limit.  So in other words

GL_MAX_VARYING_COMPONENTS = 60
GL_MAX_VERTEX_OUTPUT_COMPONENTS = 64   ... gl_PointSize  may not count against your limit

I say "may" because the spec says if you go over `GL_MAX_VARYING_COMPONENTS` then it's implementation defined whether or not it succeeds

Reply:
I see, thanks for looking into it!